docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/are-there-some-technical-services-complexities-that-might-increase-efforts.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Are there some technical complexities, that might increase efforts like: non standard services, high traffic amount, VPN?
```

Comment:
The content of the document doesn't say anything about VPN or high traffic. We need to either add it to the content or remove it from the title

Reply:
You are right, we should add to this document information regarding VPN and high traffic. I'll discuss it with Ruslan.